x-pack/plugins/task_manager/server/config.ts

```diff
@@ -128,6 +128,10 @@ export const configSchema = schema.object(
         max: DEFAULT_MAX_EPHEMERAL_REQUEST_CAPACITY,
       }),
     }),
+    /* These are not designed to be used by most users. Please use caution when changing these */
+    internal: schema.object({
+      exclude_task_types: schema.arrayOf(schema.string(), { defaultValue: [] }),
```

Comment:
Can we add a log on startup documenting that certain task types are excluded if this value is set?

Reply:
Sure!